src/clients/sidecar-openapi-specs/sidecar.openapi.yaml

```diff
@@ -1262,11 +1262,9 @@ components:
           type: string
         scram_password:
           description: The password to use when connecting to the external service.
-          maxLength: 64
+          maxLength: 1024
           minLength: 1
-          type: string
-          allOf:
-          - $ref: "#/components/schemas/Password"
+          type: object
```

Comment:
Uh, what.

Reply:
Ah, I see. The `Password` schema itself has a max length of 64, so OpenAPI is a smart-ass and says "yeah you probably don't want to use Password if you want max length longer than 64, let me give you.. object"